notebooks/vertex_genai/palm/palm_api_intro.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<p>From this descriptions, it seems you'll demonstrate both abstractive and extractive summarization, but it's not the case. I suggest either 1) remove consideration about abstractive vs extractive summarization, or 2) design a prompt showcasing extractive summarization.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/350/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>